modules/Qnite/plugin/qniteaxis.cpp

```diff
@@ -104,6 +121,22 @@ void QniteAxis::setUpperBound(qreal bound)
     qDebug() << "upper changed" << bound;
     m_upperBound = bound;
     emit upperBoundChanged();
+
+    initTicker();
+  }
+}
+
+bool QniteAxis::flip() const
```

Comment:
I don't know.. flip it's a property that holds when flipping is enabled and not a state of the axis.  that property is then forwarded to the mapper to properly compute ticks position.

A similar example is the 'clip' property of the QQuickItem: see here http://doc.qt.io/qt-5/qquickitem.html#clip-prop